Automattic/wp-calypso
# show the multi year discount line in the 2 year upsell if the annual has a free trial

## Proposed Changes

* This diff shows the "Multi-year discount" line in the 2-year upsell panel if the annual plan has a free trial and the biennial plan does not

## Testing Instructions

* To reproduce the issue, first navigate to: https://wordpress.com/checkout/jetpack/jetpack_social_advanced_yearly
* Notice that the price in the right-hand sidebar upsell for the 2 year plan does not match the correct price shown in the pricing dropdown for the 2 year plan
![Screenshot 2024-03-06 at 1 46 28 PM](https://github.com/Automattic/wp-calypso/assets/18016357/4148f515-0ddc-442d-acb3-be013bed8f32)
![Screenshot 2024-03-06 at 1 46 33 PM](https://github.com/Automattic/wp-calypso/assets/18016357/300391ff-c3d7-48c4-ac96-62e1c6ecd845)
* Now, apply this patch locally and visit: http://calypso.localhost:3000/checkout/jetpack/jetpack_social_advanced_yearly
* The sidebar upsell for two year plans should now be updated and the total price should match the price of the two year plan in the pricing dropdown
![Screenshot 2024-03-06 at 1 47 49 PM](https://github.com/Automattic/wp-calypso/assets/18016357/d3628a40-37a3-4850-b035-50580129a45c)


## Pre-merge Checklist
